packages/gestalt/src/Heading.js

```diff
@@ -18,16 +18,10 @@ const defaultHeadingLevels = {
   '400': 3,
   '500': 2,
   '600': 1,
-  'sm': 3,
-  'md': 2,
-  'lg': 1,
 };
 
 // Corresponds to the font-size design tokens
 const SIZE_SCALE = {
```

Comment:
Can this const now be removed since we no longer need to map from the old values?

Reply:
@ayeshakmaz It's mapping from string to number... do we need to keep that? Not super familiar

Reply:
I haven't tried - I would see if the string concatenation is smart enough without the conversion, otherwise using something like .Number would work. we're making a string out of this string, so hopefully that's not necessary